samples/msgext-ai-doc-compliance-checker/.localConfigs

```diff
@@ -0,0 +1,8 @@
+BOT_ID={{Azure_Bot_Id}}
```

Comment:
Mention in readme that bot id and password will be automatically generated while running sample using toolkit.

Reply:
Fixed